lib/utils/helm.go

```diff
@@ -0,0 +1,37 @@
+/*
```

Comment:
I've already seen `lib/helm/utils.go`, `lib/helm/helmutils.go` and now `lib/utils/helm.go` - any reason not to keep them all in a single package/file?